data/xml/2022.inlg.xml

```diff
@@ -0,0 +1,373 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.inlg">
+  <volume id="inlg" ingest-date="2022-11-08">
```

Comment:
Hi @xinru1414—the volume name here should be "main", not "inlg".

Reply:
yep i missed that since they are using aclpub2 format